core/src/main/scala/org/apache/spark/ui/ToolTips.scala

```diff
@@ -35,6 +35,10 @@ private[spark] object ToolTips {
 
   val OUTPUT = "Bytes and records written to Hadoop."
 
+  val STORAGE_MEMORY =
+    """Memory used / total available memory for storage of data like RDD partitions cached in
+       memory. """
+
```

Comment:
Why is this text is enclosed by triple-double-quote? It bring needless new line.
How about make the text like as follows?

```
"Memory used / total available memory for storage of data " +
  "like RDD partitions cached in memory. "
```